### ISSUE TITLE
project filter/category view - missing first level heading

### ISSUE THREAD

**Issue by IdaAkiwumi** [chnm/rrchnm2016#117]
According to WAVE, this page is missing an H1 header

https://rrchnm.org/category/projects/content/community-participation/#projects

**Comment by kimisgold** [comment]
Should be fixed in https://github.com/chnm/rrchnm-wordpress/commit/23bd3670bd681bfd836637aa3999c434e33ea92e.
